### ISSUE TITLE
Crash on fast show alerter

### ISSUE THREAD

**Issue by orcchg** [Tapadoo/Alerter#99]
```
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.ViewGroup.removeView(android.view.View)' on a null object reference
                                                                              at com.tapadoo.alerter.Alerter$1.run(Alerter.java:90)
                                                                              at android.view.ViewPropertyAnimator$AnimatorEventListener.onAnimationEnd(ViewPropertyAnimator.java:1126)
                                                                              at android.animation.ValueAnimator.endAnimation(ValueAnimator.java:1149)
                                                                              at android.animation.ValueAnimator.doAnimationFrame(ValueAnimator.java:1309)
                                                                              at android.animation.AnimationHandler.doAnimationFrame(AnimationHandler.java:146)
                                                                              at android.animation.AnimationHandler.-wrap2(AnimationHandler.java)
                                                                              at android.animation.AnimationHandler$1.doFrame(AnimationHandler.java:54)
                                                                              at android.view.Choreographer$CallbackRecord.run(Choreographer.java:925)
                                                                              at android.view.Choreographer.doCallbacks(Choreographer.java:702)
                                                                              at android.view.Choreographer.doFrame(Choreographer.java:635)
                                                                              at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:913)
                                                                              at android.os.Handler.handleCallback(Handler.java:751)
                                                                              at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                              at android.os.Looper.loop(Looper.java:154)
                                                                              at android.app.ActivityThread.main(ActivityThread.java:6692)
                                                                              at java.lang.reflect.Method.invoke(Native Method)
                                                                              at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1468)
                                                                              at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1358)
```

maybe just check for nullability and skip showing alert?

actually, here:

```
@NonNull
    private static Runnable getRemoveViewRunnable(final Alert childView) {
        return new Runnable() {
            @Override
            public void run() {
                ((ViewGroup) childView.getParent()).removeView(childView);
            }
        };
    }
```

**Comment by kpmmmurphy** [comment]
Hi @orcchg, thanks for flagging this. I'd be happy to review your pull-request once it's ready.

**Comment by kpmmmurphy** [comment]
I've implemented a fix for this in version 2.0.3.
